readthedocs/settings/base.py

```diff
@@ -41,6 +41,7 @@ class CommunityBaseSettings(Settings):
     PUBLIC_DOMAIN_USES_HTTPS = False
     USE_SUBDOMAIN = False
     PUBLIC_API_URL = 'https://{}'.format(PRODUCTION_DOMAIN)
+    RTD_PROXIED_API_URL = PUBLIC_API_URL
```

Comment:
It would be good to add a comment in the code for this setting since it's not easy to realize what it is for and what's the difference with PUBLIC_API_URL

Reply:
Done